app/javascript/packs/application.ts

```diff
@@ -2,3 +2,9 @@ import { accordion, banner, skipnav } from 'identity-style-guide';
 
 const components = [accordion, banner, skipnav];
 components.forEach((component) => component.on());
+const mainContent = document.getElementById('main-content');
+document.querySelector('.usa-skipnav')?.addEventListener('click', (event) => {
+  event.preventDefault();
+  mainContent?.focus();
```

Comment:
Is the `focus` necessary, if this is already happening in the base USWDS implementation?

https://github.com/uswds/uswds/blob/e75f3f347170ebfe472bba9719935a4f5ba38b69/packages/usa-skipnav/src/index.js#L20

```suggestion
```

Reply:
No it isn't. Good call! Thanks @aduth